### ISSUE TITLE
Activation Function and Loss Function in Neural Network:

### ISSUE THREAD

**Issue by Anshika14528** [Recode-Hive/machine-learning-repos#18]
Explaining about activation and loss functions in neural network.

I agree to follow this project's Code of Conduct
I'm a GSSoC'24 contributor
I want to work on this issue

**Comment by lassmara** [comment]
Please assign me this issue.

**Comment by itssiddhantjain** [comment]
Hello @sanjay-kv , please assign this issue to me as I have already worked on this kind of problem in the past.
